source/SH3/graphics/texture.cpp

```diff
@@ -0,0 +1,36 @@
+/** @file
+ *
+ *  Implementation of texture.hpp
+ *
+ *  \copyright 2016  Palm Studios
+ *
+ *  \note
+ *
+ *  \date 2/1/2017
+ *
+ *  \author Jesse Buhagiar
+ */
+#include <SH3/graphics/texture.hpp>
+#include <SH3/system/log.hpp>
+
+using namespace sh3_graphics;
+
+/**
```

Comment:
Documentation should be in the header, because you're documenting the interface.